vendor/vendor.json

```diff
@@ -14,10 +14,6 @@
 			"revision": "50fc57a35b373285b4d46db88c19dc86d605724b",
 			"revisionTime": "2017-03-03T23:35:56Z"
 		},
-		{
```

Comment:
Assuming this was intentional?

Reply:
See the first Travis failure in the first commit - it would otherwise fail... I guess it's not great to stick such change to this PR, but I wanted to see green 😜